src/components/views/elements/TooltipButton.tsx

```diff
@@ -19,8 +19,16 @@ import React from 'react';
 import * as sdk from '../../../index';
 import {replaceableComponent} from "../../../utils/replaceableComponent";
 
+interface IProps {
+    helpText: string;
+}
+
+interface IState {
+    hover: boolean;
+}
+
 @replaceableComponent("views.elements.TooltipButton")
-export default class TooltipButton extends React.Component {
+export default class TooltipButton extends React.Component<IProps, IState> {
     state = {
```

Comment:
Move this into the constructor and use `this.state = { ... }` to match our more usual way of doing this.

Reply:
We have a bunch of these too fwiw

Reply:
Well, I was instructed by @turt2live to convert one over in [another PR](https://github.com/matrix-org/matrix-react-sdk/pull/6076#discussion_r636394699)... I don't have a strong opinion myself, but we should perhaps decide collectively whether we care or not.